src/dispatch.rs

```diff
@@ -68,15 +68,16 @@ impl AsyncDispatcher {
     }
 
     /// Waits for all the systems to finish
-    /// and executes thread local systems (if there
-    /// are any).
     pub fn wait(&mut self) {
         self.signal
             .take()
             .expect(ERR_NO_DISPATCH)
             .wait()
             .expect("The worker thread may have panicked");
+    }
```

Comment:
Why would I want to block for some amount of time but not execute the thread local systems?